app/src/main/java/pl/edu/zut/mad/appzut/utils/DateUtils.java

```diff
@@ -83,4 +98,42 @@ public static Date convertStringToDate(@NonNull String date) {
             return null;
         }
     }
+
+    public static Date convertDateTimeToDate(@NonNull DateTime dateTime) {
+        return CalendarHelper.convertDateTimeToDate(dateTime);
```

Comment:
Is this wrapper needed? And if you're keeping it add comment that this method converts only date and time is zeroed.